src/cmd/build.js

```diff
@@ -103,15 +111,49 @@ async function defaultPackageCreator({
   let packageName = safeFileName(
     `${extensionName}-${manifestData.version}.zip`);
   let extensionPath = path.join(artifactsDir, packageName);
-  let stream = createWriteStream(extensionPath);
 
-  stream.write(buffer, () => stream.end());
+  let destinationFileExists = await fs.stat(extensionPath)
```

Comment:
The beauty about [await](https://ponyfoo.com/articles/understanding-javascript-async-await) is that you don't need to construct a promise chain! This block can be rewritten as:
````javascript
let destStat;
try {
  destStat = await fs.stat(extensionPath);
} catch (error) {
  // Here you will need to check for a known error code meaning 'destination does not exist'.
  if (error.code === 'something') {
    destStat = false;
  } else {
    throw error;
  }
}
const destinationFileExists = destStat && desStat.isFile();
````